requirements.txt

```diff
@@ -0,0 +1,2 @@
+ops==1.1.0
+git+https://github.com/johnsca/resource-oci-image@25b480913c250b07be2821c05fd47e324ff75ae8
```

Comment:
Why are we using the johnsca repo and not the [juju-solutions](https://github.com/juju-solutions/resource-oci-image) one?

Reply:
It was moved, and the johnsca repo just redirects now.  It would be better to switch to the one in juju-solutions.  Though we should maybe also consider releasing it to PyPI and using a proper release process and version control; I was sort of waiting on a resolution on the charm libraries discussion.